### ISSUE TITLE
iOS 14 bug: trending items cannot be clicked

### ISSUE THREAD

**Issue by mmdock** [Giphy/giphy-ios-sdk-ui-example#90]
There is a number of trending items at the top of GiphyViewController.  As a user I kinda expect these to be selectable and auto search for this, but nothing happens when selected

setup:

```
                let giphy = GiphyViewController()
                GiphyViewController.trayHeightMultiplier = 0.6
                giphy.setAPIKey(Application.settings.giphySDKKey)
                giphy.theme = GPHTheme(type: .light)
                giphy.mediaTypeConfig = [.gifs, .recents, .stickers, .emoji, .text]
                giphy.showConfirmationScreen = false
                giphy.shouldLocalizeSearch = true
                giphy.delegate = self
                giphy.dimBackground = true
                
                giphy.modalPresentationStyle = .overCurrentContext
                
                self.present(giphy, animated: true, completion: nil)
```

Note: I confirmed this occurs when building for iOS 14 devices(physical and simulator) with Xcode Beta, but pre-iOS 14 simulated devices still works fine
Giphy version: - Giphy (2.0.1):

**Comment by cgmaier** [comment]
thanks for flagging this, @mmdock !

**Comment by cgmaier** [comment]
hey @mmdock this should be fixed in the latest release 2.0.2
